spec/openapi.yaml

```
@@ -2791,6 +2791,10 @@ components:
           description: Order reference for business sales. Only available if business model is `B2B`
           type: string
           maxLength: 255
+        tax_region:
+          description: The region that has been used for tax calculations
```

Appending a suffix regarding the format would make it consistent with some other fields like `countryCode`
```suggestion
          description: The region that has been used for tax calculations (ISO 3166-1 alpha-2)
```